modules/configure-additional-cas.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * dir/filename.adoc
+:_module-type: PROCEDURE
+[id="configure-additional-cas_{context}"]
+= Configuring additional CAs
+
+To add custom CAs:
+
+.Procedure
+
+. Download the link:https://help.stackrox.com/downloads/927d7f5b23e88089b5545450cbfa7cc1/ca-setup.sh[_ca-setup.sh_] script.
```

Comment:
@IAmATeaPot418 Where are we planning to host this file? 
- And other files in `src/files` folders from our StackRox docs repository?